templates/deno/.vscode/resolve_npm_imports.json

```diff
@@ -9,9 +9,9 @@
   "imports": {
     "// `@remix-run/deno` code is already a Deno module, so just get types for it directly from `node_modules/`": "",
     "@remix-run/deno": "../node_modules/@remix-run/deno/index.ts",
-    "@remix-run/server-runtime": "https://esm.sh/@remix-run/server-runtime@1.7.2",
-    "@remix-run/dev/server-build": "https://esm.sh/@remix-run/dev@1.6.4/server-build",
-    "@remix-run/react": "https://esm.sh/@remix-run/react@1.6.4",
+    "@remix-run/server-runtime": "https://esm.sh/@remix-run/server-runtime@latest",
+    "@remix-run/dev/server-build": "https://esm.sh/@remix-run/dev@latest/server-build",
+    "@remix-run/react": "https://esm.sh/@remix-run/react@latest",
```

Comment:
Shouldn't these be the exact version we have in `package.json` (ie currently `1.8.2`)?

People will have to update these consciously when also updating `package.json` versions.
People will also have errors when v2 will be released otherwise